packages/block-library/src/template-part/editor.scss

```diff
@@ -1,82 +1,36 @@
+.block-editor-template-part__selection-modal {
+	@include break-medium() {
+		width: $break-medium - $grid-unit-20 * 2;
+	}
 
-.wp-block-template-part__placeholder-preview-dropdown-content,
-.wp-block-template-part__preview-dropdown-content {
-	.components-popover__content {
-		min-width: 320px;
+	.components-modal__content {
 		padding: 0;
 	}
 }
 
-.wp-block-template-part__selection-preview-search-form {
-	border-bottom: $border-width solid $gray-300;
-}
-
-.wp-block-template-part__selection-preview-container {
-	background: $white;
-	padding-bottom: 16px;
-
-	.wp-block-template-part__selection-preview-item {
-		border-radius: $radius-block-ui;
-		cursor: pointer;
-		margin-top: $grid-unit-20;
-		transition: all 0.05s ease-in-out;
-		border: $border-width solid $gray-300;
-		width: 100%;
-		background-color: $white;
-
-		&:hover {
-			border: $border-width solid var(--wp-admin-theme-color);
-		}
-
-		&:focus {
-			box-shadow: inset 0 0 0 1px $white, 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
-
-			// Windows High Contrast mode will show this outline, but not the box-shadow.
-			outline: 2px solid transparent;
-		}
-
-		&.is-placeholder {
-			min-height: 100px;
-		}
-
-		.block-editor-block-preview__container {
-			border-bottom: $border-width solid $gray-300;
-		}
-	}
-
-	.wp-block-template-part__selection-preview-item-title {
-		padding: $grid-unit-05;
-		font-size: $helptext-font-size;
-		text-align: left;
-	}
-
-	.wp-block-template-part__selection-panel-group-header {
-		padding: $grid-unit-20 $grid-unit-20 0;
-	}
+.block-library-template-part__selection-content .block-editor-block-patterns-list {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
```

Comment:
I think it would look nicer if we adjusted the `columns` per viewport widths as we do in Pattern Explorer.

Reply:
Yeah, I also feel this logic should be a prop of the component instead of doing this in CSS.